File: Assignment 5/Q3.c
/* Write a Program which Accept Number from user and print its numbers line.
   Input : 4
   Output : -4  -3  -2  -1  0   1   2   3   4
*/

#include<stdio.h>

void Display(int iNo)
{
    int iCount = 0;
    for(iCount=-iNo; iCount<=iNo;  iCount++)
    {
        printf("%d\t",iCount);
    }
}

int main()
{
    int iValue = 0;

    printf("Enter Number : ");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}

File: Assignment 17/Q3.java
// Write a Program to find the Maximum of Two Numbers

class Logic
{
    void findMax(int a, int b)
    {
        if(a > b)
        {
            System.out.println("Maximum Number is : "+ a);
        }
        else
        {
            System.out.println("Maximum Number is : "+ b);
        }
    }
}

class Q3
{
    public static void main(String args[])
    {
        Logic obj = new Logic();
        obj.findMax(20,15);
    }
}